htgettoken.spec

```diff
@@ -4,126 +4,67 @@ Summary: Get OIDC bearer tokens by interacting with Hashicorp vault
 Name: htgettoken
 Version: 1.16
 Release: 1%{?dist}
-License: BSD
-Group: Applications/System
+
+License: BSD-3-Clause
 URL: https://github.com/fermitools/htgettoken
+
+BuildArch: noarch
+Prefix: %{_prefix}
+
 # download with:
 # $ curl -o htgettoken-%{version}.tar.gz \
 #    https://codeload.github.com/fermitools/htgettoken/tar.gz/%{version}
 Source0: %{name}-%{version}.tar.gz
-# recreate this with make-downloads
-Source1: %{name}-downloads-%{downloads_version}.tar.gz
-BuildRequires: python3-pip
-BuildRequires: python3-devel
-# swig and openssl-devel are needed to prevent an M2Crypto problem with
-#   OpenSSL 1.1
-BuildRequires: gcc
-BuildRequires: krb5-devel
-BuildRequires: swig
-BuildRequires: openssl-devel
-
-# Needed by httokendecode
+
+# rpmbuild dependencies
+BuildRequires: python-rpm-macros
+
+# -- Package: htgettoken
+
+# /usr/bin/htgettoken:
+Requires: python%{python3_pkgversion}-gssapi
+Requires: python%{python3_pkgversion}-paramiko
+Requires: python%{python3_pkgversion}-urllib3
+# /usr/bin/httokendecode:
 Requires: jq
+%if 0%{?rhel} && 0%{?rhel} >= 8
+Recommends: scitokens-cpp
+%endif
```

Comment:
Why is rhel 7 excluded from this Recommends?  Is it not supported there yet?

Reply:
Exactly, the old `rpmbuild` on EL7 can't handle it.